types/wordpress__blocks/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for @wordpress/blocks 9.1
+// Type definitions for @wordpress/blocks 11.0
```

Comment:
[👍 Looks good for the upgraded dep](https://unpkg.com/browse/@wordpress/blocks@11.0.0/package.json)